### PR TITLE
fix: improve local development configuration instructions

### PR DESCRIPTION
**Pull request by overbit**
## PR Details

Minor improvement on the Getting started with local development

## PR Checklist
- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,9 @@ cd amplication
 2. Run the [setup script](https://github.com/amplication/amplication/blob/master/script/setup.ts), which takes care of installing dependencies, building packages and ensuring your workspace is dev-ready.
 
 ```bash
+# Due to ts-node dependency
+npm install -g ts-node # or just npm install
```
Comment (yuval-hazaz) [inline README.md:100]: I think this better be added to the prerequisite above (line 81). WDYT?

Comment (overbit) [reply README.md:100]: Great point. I'll do asap.

Comment (yuval-hazaz) [reply README.md:100]: @overbit I don't see the change

Comment (overbit) [reply README.md]: @yuval-hazaz forgot to push from my old laptop :-S

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**